### ISSUE TITLE
Override pagination template with custom view

### ISSUE THREAD

**Issue by dstollie** [matteodem/meteor-easy-search#217]
I cant figure out how to override the current pagination template with my own. Im using foundation and it has it's own pagination component which i want to use.

**Comment by matteodem** [comment]
Is renderControl not sufficient? see http://matteodem.github.io/meteor-easy-search/docs/blaze-components/

**Comment by renato0307** [comment]
Hi @matteodem,

I'm new to meteor and easy search.

Can you please provide an example?

Thank you in advance,
Renato

**Comment by renato0307** [comment]
I've created a dummy template:
```html
<template name="searchPagination">
<strong>X{{text}}X</strong>
</template>
```

And added the renderControl parameter to the esPagination component:
```html
{{> esPagination index="documents" renderControl=Template.searchPagination}}
```

But the new template is not used:

![captura de ecra 2015-07-26 as 22 37 23](https://cloud.githubusercontent.com/assets/2320628/8896033/eecdebce-33e6-11e5-91d8-4152f366b431.png)

Thank you.

**Comment by matteodem** [comment]
This is fixed in the latest release 1.6.2, feel free to re-open if you think not.
